roles/hosted_engine_setup/tasks/create_target_vm/03_hosted_engine_final_tasks.yml

```diff
@@ -162,8 +162,6 @@
     command: dd bs=1M count=1024 oflag=direct if=/dev/zero of="{{ he_metadata_disk_path }}"
     environment: "{{ he_cmd_lang }}"
     become: true
-    become_user: vdsm
-    become_method: sudo
```

Comment:
Who will be the owner of the file, then? Please explain in a code comment or the commit message why we needed to be vdsm and now we do not. Same for below.

Reply:
Why are you zeroing 1g in this volume?

If you want to allocate space, use:

   fallocate --length ... /path/to/file

If you want to clean up a block device, use:

    blkdiscard --zeroout /path/to/device
    blkdiscard /path/to/device

Reply:
> Why are you zeroing 1g in this volume?
> 
> If you want to allocate space, use:
> 
> fallocate --length ... /path/to/file
> 
> If you want to clean up a block device, use:
> 
> ```
> blkdiscard --zeroout /path/to/device
> blkdiscard /path/to/device
> ```

It's not really related to this PR and I don't know the historical reason behind that, I'll check that, thanks.